app/lib/actions/remote_execution/run_hosts_job.rb

```diff
@@ -71,6 +77,16 @@ def determine_proxy(template_invocation, host, load_balancer)
 
         proxy
       end
+
+      def provider_job_class(provider)
+        case provider
+        when 'SSH'
+          RunSSHJob
```

Comment:
Sorry, I'm not sure what you mean. This method provides a matching between the `provider_type` attribute of a template and the job class to be triggered. How would I put this matching in the provider class?